docker/prometheus/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.8'
+
+  services:
```

Comment:
I would suggest adding these containers to the `docker-compose.yml` file at the root of the project, perhaps in a profile that can be enabled or disabled specifically. Then `prometheus.yml` could target `erddap:8080` by default and users wouldn't have to edit any files to get monitoring on their local stack.